apps/coscrad-frontend/src/components/higher-order-components/use-loadable-categorizables.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+    CategorizableCompositeIdentifier,
+    CategorizableType,
+    IBaseViewModel,
+    ICategorizableDetailQueryResult,
+    ICategorizableIndexQueryResult,
+} from '@coscrad/api-interfaces';
+import { isNull, isNullOrUndefined } from '@coscrad/validation-constraints';
+import { useSelector } from 'react-redux';
+import { useAppDispatch } from '../../app/hooks';
+import { RootState } from '../../store';
+import { ILoadable } from '../../store/slices/interfaces/loadable.interface';
+import { fetchFreshState } from '../../store/slices/utils/fetch-fresh-state';
+
+// Hack alert- we need a mapping layer to give us the plurals for our redux slice names
```

Comment:
This relates to the configurable content as well. Try to clean this up now if possible.

Reply:
[Here](https://www.pivotaltracker.com/story/show/184310526) is a story for that work.